src/lambda_codebase/initial_commit/bootstrap_repository/adf-build/shared/python/organizations.py

```diff
@@ -28,16 +32,36 @@ class Organizations:  # pylint: disable=R0904
 
     _config = Config(retries=dict(max_attempts=30))
 
-    def __init__(self, role, account_id=None):
-        self.client = role.client(
-            'organizations',
-            config=Organizations._config
+    def __init__(
+        self, role=None, account_id=None, org_client=None, tagging_client=None
+    ):
+        if role:
+            LOGGER.warning(
+                "Using a role in the organizations client is being deprecated. Please provide the relevant clients to remove this warning"
```

Comment:
This line is quite lengthy, could you split the message on two lines instead?
Could you also clearly state at the beginning that this is a "Deprecation warning:"?

Reply:
```suggestion
                "Deprecation warning: Using a role in the organizations client is being "
                "deprecated. Please provide the relevant clients to remove this warning"
```